Dockerfile-bbchem

```diff
@@ -15,17 +15,48 @@ RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
 ADD 	. /src
 RUN 	mvn clean && \
         mvn compile war:war -P ${CHEMISTRY_PACKAGE}
-RUN     mv target/acas*.war $CATALINA_HOME/webapps/acas.war && \
-        mv target/acas* $CATALINA_HOME/webapps/acas/
 
-FROM 	${TOMCAT_IMAGE} as build
-COPY 	--chown=runner:runner --from=compile /usr/local/tomcat/webapps/acas.war $CATALINA_HOME/webapps/acas.war
-COPY 	--chown=runner:runner --from=compile /usr/local/tomcat/webapps/acas $CATALINA_HOME/webapps/acas
+FROM tomcat:9.0.58-jre8-openjdk-slim-buster
```

Comment:
Most of the stuff below here is stuff that was in `mcneilco/tomcat-maven` repo previously.

You'll also see this repeated in most of the docker files that we are keeping around to support chef based docker builds which cannot use multi-stage builds.  Eventually we'll only keep one, multistage, Dockerfile